UnitySDK/Assets/ML-Agents/Scripts/InferenceBrain/TensorGenerator.cs

```diff
@@ -1,106 +1,106 @@
-            _dict[TensorNames.VectorObservationPlacholder] = new VectorObservationGenerator();
-            _dict[TensorNames.RecurrentInPlaceholder] = new RecurrentInputGenerator();
-            
-            #if ENABLE_BARRACUDA
-            Barracuda.Model model = (Barracuda.Model) barracudaModel;
-            for (var i = 0; i < model?.memories.Length; i++)
-            {
-                _dict[model.memories[i].input] = new BarracudaRecurrentInputGenerator(i);
-            }
-            #endif
-            
-            _dict[TensorNames.PreviousActionPlaceholder] = new PreviousActionInputGenerator();
-            _dict[TensorNames.ActionMaskPlaceholder] = new ActionMaskInputGenerator();
-            _dict[TensorNames.RandomNormalEpsilonPlaceholder] = new RandomNormalInputGenerator(seed);
-            if (bp.cameraResolutions != null)
-            {
-                for (var visIndex = 0;
-                    visIndex < bp.cameraResolutions.Length;
-                    visIndex++)
-                {
-                    var index = visIndex;
-                    var bw = bp.cameraResolutions[visIndex].blackAndWhite;
-                    _dict[TensorNames.VisualObservationPlaceholderPrefix + visIndex] = new
-                            VisualObservationInputGenerator(index, bw);
-                }
-            }
-
-            // Generators for Outputs
-            _dict[TensorNames.ActionOutput] = new BiDimensionalOutputGenerator();
-            _dict[TensorNames.RecurrentOutput] = new BiDimensionalOutputGenerator();
-            _dict[TensorNames.ValueEstimateOutput] = new BiDimensionalOutputGenerator();
-        }
-
-        /// <summary>
-        /// Populates the data of the tensor inputs given the data contained in the current batch
-        /// of agents.
-        /// </summary>
-        /// <param name="tensors"> Enumerable of tensors that will be modified.</param>
-        /// <param name="currentBatchSize"> The number of agents present in the current batch
-        /// </param>
-        /// <param name="agentInfos"> Dictionary of Agent to AgentInfo that contains the
-        /// data that will be used to modify the tensors</param>
-        /// <exception cref="UnityAgentsException"> One of the tensor does not have an
-        /// associated generator.</exception>
-        public void GenerateTensors(IEnumerable<Tensor> tensors, 
-            int currentBatchSize, 
-            Dictionary<Agent, AgentInfo> agentInfos)
-        {
-            foreach (var tensor in tensors)
-            {
-                if (!_dict.ContainsKey(tensor.Name))
-                {
-                    throw new UnityAgentsException(
-                        "Unknow tensor expected as input : " + tensor.Name);
-                }
-                _dict[tensor.Name].Generate(tensor, currentBatchSize, agentInfos);
-            }
-        }
-    }
-}
+﻿#define ENABLE_BARRACUDA
```

Comment:
Why this script get changed?

Reply:
Oh I see, just saw your comment.

Reply:
Yeah, it was using different line endings. You can change the diff settings to ignore whitespace and they're identical: 
![image](https://user-images.githubusercontent.com/6877802/60468520-bf08a200-9c0e-11e9-8bfb-867612917007.png)